mantidproject/mantid
# LoadMcStas single, multiple and sum of scattering output

Previously the `LoadMcStas` algorithm only returned the sum of single and multiple scattering eventdata. 
This change means the algorithm now outputs both the single and multiple scattering data as well as the sum of the two. 
 
**To test:**

Run the LoadMcStas algorithm
*Note: The required file for the below is in the DocTest data directory*
```
test = LoadMcStas('mcstas_event_hist.h5')
```
* Observe that there are now 3 `EventData` workspaces: `EventData_test, EventData_test_1, EventData_test_2`
* `EventData_test_1` and `_2` should sum to `EventData_test`.
* When showing the instrument on these workspaces you should see that `EventData_test` has two banks, in `EventData_test_1`, `EventData_test_2` contain one of these banks each.


Fixes #20369

**Release Notes** 
Update to framework page in release notes [here](https://github.com/mantidproject/mantid/blob/ec789a914d7075d8b3c679a5a771c10abaafd377/docs/source/release/v3.12.0/framework.rst)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
